src/PRDevice.h

```diff
@@ -95,7 +95,7 @@ class PRDevice
     PRResult PRLEDRGBFade(PRLEDRGB * pLED, uint32_t fadeColor, uint16_t fadeRate);
     PRResult PRLEDRGBFadeColor(PRLEDRGB * pLED, uint32_t fadeColor);
 
-    int GetVersionInfo(uint16_t *verPtr, uint16_t *revPtr, uint16_t *combinedPtr);
+    int GetVersionInfo(uint16_t *verPtr, uint16_t *revPtr, uint32_t *combinedPtr);
```

Comment:
I don't know if it will be necessary to update other programs making use of this API, but the value assigned to `combinedPtr` is 32-bits, so the passed pointer should be a for a 32-bit variable.